### PR TITLE
[Docs] Update contributing, conduct, readme. Close #349

### LINKED ISSUE #349: Revise documentation, update anything that needs it
Updates on: 

- schedule
- contributing
- readme

### PR DESCRIPTION
**Pull request by jessesquires**
Update contributing, conduct, readme. Close #349

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,27 +6,35 @@ First and foremost, please review and abide by [our code of conduct](https://git
...
+1. Open an issue on GitHub. Set the title to the newsletter issue number and date. 
+**Example:** `[100] Issue #100 - Jan 04, 2018`
+1. Assign it to yourself. 
+1. Add the [`current issue` and `full issue notes`](https://github.com/SwiftWeekly/swiftweekly.github.io/issues?utf8=✓&q=is%3Aissue+label%3A%22current+issue%22+label%3A%22full+issue+notes%22+) labels.  
+1. We use this throughout the week to collect notes and make it easy for external contributors to contribute content. We also use this issue to collaborate with the other [Swift Weekly writers](https://swiftweekly.github.io/authors/) and troubleshoot any problems that come up.
 1. Write the issue in `_drafts/` throughout the week. We recommend writing a little each day to make it easier. 
-1. For style and content, follow the example set by previous issues.
-1. **For code snippets, use jekyll highlight blocks:** `{% highlight swift %} /*code*/ {% endhighlight %}`
-1. See the handy tips below!
+1. For style and content, set the guide below. Also, follow the example set by previous issues.
```
Comment (BasThomas) [inline CONTRIBUTING.md]: Typo: "see* the guide below."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,58 +88,55 @@ We encourage all writes to use their own voice and express their own personality
...
+* The Core Team or broader Swift team is highly engaged in the discussion.
 
-> **Note:** This does not mean that discussions without these characteristics aren't worthwhile. These are simply a few guidelines to evaluate discussions.
+> 💡 **Note:** This does not mean that discussions without these characteristics are not worth mentioning. These are simply a few guidelines.
 
-# Publishing an issue
+# 📰 Publishing an issue
 
-:warning: We publish **every Thursday**. :warning:
+> 💡 **Note:** We publish **every Thursday**.
```
Comment (BasThomas) [inline CONTRIBUTING.md]: "every other Thursday"?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -77,58 +88,55 @@ We encourage all writes to use their own voice and express their own personality
...
 Tweets from [@swiftlybrief](https://twitter.com/swiftlybrief) are automated via [IFTTT](https://ifttt.com). 
 
-### Continuous integration
+## 🎢 Continuous integration
 
 We use [travis-ci](https://travis-ci.org/SwiftWeekly/swiftweekly.github.io), [danger](http://danger.systems), and [proselint](http://proselint.com) (via [danger-prose](https://github.com/dbgrandi/danger-prose)). This ensures:
 
-- We're always pushing valid markdown and `jekyll` is happy
+- We're always pushing valid markdown and `jekyll` can build the site.
 - We write elegant prose — no grammar/spelling mistakes, and no offensive or unwelcoming language. See the [`Dangerfile`](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/Dangerfile) for more details.
```
Comment (BasThomas) [inline CONTRIBUTING.md]: Do we want to cross this out as well as it has been disabled for now?

File: README.md [hunk 4]
```diff
@@ -2,15 +2,13 @@
...
 
 <img src="https://raw.githubusercontent.com/SwiftWeekly/swiftweekly.github.io/master/img/logo.png" width="250"/>
 
 ## About
 
-This newsletter was originally started by [Jesse Squires](https://github.com/jessesquires), and hosted on his blog at [jessesquires.com](http://www.jessesquires.com). After issue #4, this site was created to provide the newsletter with a proper, dedicated home. All previous issues were [migrated](http://www.jessesquires.com/new-weekly-brief/), and it is now completely open for the community. Contributions are welcomed and encouraged!
+This newsletter was originally started by [Jesse Squires](https://github.com/jessesquires), and hosted on his blog at [jessesquires.com](http://www.jessesquires.com). After issue #4, this site was created to provide the newsletter with a proper, dedicated home. All previous issues were [migrated](http://www.jessesquires.com/new-weekly-brief/), and it is now completely open for the community. Now [Bas Broek](https://github.com/BasThomas) is currently leading the project. Contributions are welcomed and encouraged!
```
Comment (BasThomas) [inline README.md]: "Now Bas Broek is currently" -> "Currently, Bas Broek is" would prevent the duplication.

File: README.md [hunk 5]
```diff
@@ -52,11 +50,11 @@ $ bundle exec jekyll serve --future --drafts --watch
 
 ## Contributing
 
-For content suggestions, fixes, or corrections — please see [`CONTRIBUTING.md`](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).
+To contribute, please read [`CONTRIBUTING.md`](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).
```
Comment (BasThomas) [inline README.md]: To be consistent with "Please read our Code of Conduct [..]", let's make this "To contribute, please read **the** Contributing.md"

File: CONTRIBUTING.md (hunk 2 above)
Comment (jessesquires) [reply CONTRIBUTING.md]: oops, right!

File: CONTRIBUTING.md (hunk 3 above)
Comment (jessesquires) [reply CONTRIBUTING.md]: I'm going to work on a PR to try to re-enable

File: README.md (hunk 5 above)
Comment (jessesquires) [reply README.md]: 👌

**Pull request merged**